### PR TITLE
[zh-cn] Add horizontal-pod-autoscaler-v2.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fixed: #34733 -HPA-v2
Add horizontal-pod-autoscaler-v2.md chinese version

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 1]
```diff
@@ -0,0 +1,2359 @@
...
+  
+  <!--
+  - **metrics.external** (ExternalMetricSource)
+
+    external refers to a global metric that is not associated with any Kubernetes object. It allows autoscaling based on information coming from components running outside of cluster (for example length of queue in cloud messaging service, or QPS from loadbalancer running outside of cluster).
+  -->
+  
+  - **metrics.external** (ExternalMetricSource)
+
+    external 指的是与任何 Kubernetes 对象无关的全局指标。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:497]:
```suggestion
    external 指的是不与任何 Kubernetes 对象关联的全局指标。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 2]
```diff
@@ -0,0 +1,2359 @@
...
+    它允许基于来自集群外部运行的组件（例如云消息服务中的队列长度，或来自运行在集群外部的负载均衡器的 QPS）的信息进行自动扩缩容。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:498]:
```suggestion
    这一字段允许基于来自集群外部运行的组件（例如云消息服务中的队列长度，或来自运行在集群外部的负载均衡器的 QPS）的信息进行自动扩缩容。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 3]
```diff
@@ -0,0 +1,2359 @@
...
+    <!--
+    *ExternalMetricSource indicates how to scale on a metric not associated with any Kubernetes object (for example length of queue in cloud messaging service, or QPS from loadbalancer running outside of cluster).*
+
+    - **metrics.external.metric** (MetricIdentifier), required
+
+      metric identifies the target metric by name and selector
+    -->
+    
+    **ExternalMetricSource 指示如何扩缩与任何 Kubernetes 对象无关的指标
+    （例如云消息传递服务中的队列长度，或来自集群外部运行的负载均衡器的 QPS）。**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:511]:
```suggestion
    ExternalMetricSource 指示如何基于 Kubernetes 对象无关的指标
    （例如云消息传递服务中的队列长度，或来自集群外部运行的负载均衡器的 QPS）执行扩缩操作。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 4]
```diff
@@ -0,0 +1,2359 @@
...
+
+    - **metrics.external.metric** (MetricIdentifier)，必需
+
+      metric 通过名称和选择器标识目标指标。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:515]:
```suggestion
      metric 通过名称和选择算符识别目标指标。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 5]
```diff
@@ -0,0 +1,2359 @@
...
+      
+      <!--
+      - **metrics.external.metric.selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)
+
+        selector is the string-encoded form of a standard kubernetes label selector for the given metric When set, it is passed as an additional parameter to the metrics server for more specific metrics scoping. When unset, just the metricName will be used to gather metrics.
+      -->
+      
+      - **metrics.external.metric.selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)
+
+        selector 是给定指标的标准 Kubernetes 标签选择器的字符串编码形式。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:541]:
```suggestion
        selector 是给定指标的标准 Kubernetes 标签选择算符的字符串编码形式。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 6]
```diff
@@ -0,0 +1,2359 @@
...
+      - **metrics.external.target.type** (string), required
+
+        type represents whether the metric type is Utilization, Value, or AverageValue
+      -->
+      
+      **MetricTarget 定义特定指标的目标值、平均值或平均利用率**
+
+      - **metrics.external.target.type** (string)，必需
+
+        type 表示指标类型是 Utilization、Value 或 AverageValue。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:569]:
```suggestion
        type 表示指标类型是 `Utilization`、`Value` 或 `AverageValue`。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 7]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **metrics.external.target.averageUtilization** (int32)
+
+        averageUtilization is the target value of the average of the resource metric across all relevant pods, represented as a percentage of the requested value of the resource for the pods. Currently only valid for Resource metric source type
+      -->
+      
+      - **metrics.external.target.averageUtilization** (int32)
+
+        averageUtilization 是 Pod 所有相关的资源指标平均值的目标值，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:579]:
```suggestion
        averageUtilization 是跨所有相关 Pod 得到的资源指标均值的目标值，
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 8]
```diff
@@ -0,0 +1,2359 @@
...
+        表示为 Pod 资源请求值的百分比。目前仅对资源指标源类型有效。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:580]:
```suggestion
        表示为 Pod 资源请求值的百分比。目前仅对 “Resource” 指标源类型有效。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 9]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **metrics.external.target.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue is the target value of the average of the metric across all relevant pods (as a quantity)
+      -->
+      
+      - **metrics.external.target.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue 是 Pod 所有相关指标平均值的目标值（作为数量）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:590]:
```suggestion
        averageValue 是跨所有相关 Pod 得到的指标均值的目标值（以数量形式给出）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 10]
```diff
@@ -0,0 +1,2359 @@
...
+
+      <!--
+      - **metrics.external.target.value** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        value is the target value of the metric (as a quantity).
+      -->
+      
+      - **metrics.external.target.value** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        value 是指标的目标值（作为数量）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:600]:
```suggestion
        value 是指标的目标值（以数量形式给出）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 11]
```diff
@@ -0,0 +1,2359 @@
...
+
+  <!--
+  - **metrics.object** (ObjectMetricSource)
+
+    object refers to a metric describing a single kubernetes object (for example, hits-per-second on an Ingress object).
+  -->
+  
+  - **metrics.object** (ObjectMetricSource)
+
+    object 是指描述单个 Kubernetes 对象的指标（例如，Ingress 对象上的每秒点击数）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:610]:
```suggestion
    object 是指描述单个 Kubernetes 对象的指标（例如，Ingress 对象上的 `hits-per-second`）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 12]
```diff
@@ -0,0 +1,2359 @@
...
+  
+    <!--
+    *ObjectMetricSource indicates how to scale on a metric describing a kubernetes object (for example, hits-per-second on an Ingress object).*
+
+    - **metrics.object.describedObject** (CrossVersionObjectReference), required
+
+      describedObject specifies the descriptions of a object,such as kind,name apiVersion
+    -->
+    
+    **ObjectMetricSource 表示如何根据描述 Kubernetes 对象的指标进行扩缩容（例如，Ingress 对象的每秒点击数）**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:622]:
```suggestion
    **ObjectMetricSource 表示如何根据描述 Kubernetes 对象的指标进行扩缩容（例如，Ingress 对象的 `hits-per-second`）**
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 13]
```diff
@@ -0,0 +1,2359 @@
...
+
+    - **metrics.object.describedObject** (CrossVersionObjectReference)，必需
+
+      describeObject 表示对象的描述，如对象的 kind、name、apiVersion。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:626]:
```suggestion
      describeObject 表示对象的描述，如对象的 `kind`、`name`、`apiVersion`。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 14]
```diff
@@ -0,0 +1,2359 @@
...
+      - **metrics.object.describedObject.kind** (string), required
+
+        Kind of the referent; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"
+      -->
+      
+      **CrossVersionObjectReference 包含足够的信息来让你识别所引用的资源。**
+
+      - **metrics.object.describedObject.kind** (string)，必需
+      
+        被引用对象的类型；更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:642]:
```suggestion
        被引用对象的类别；更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 15]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **metrics.object.describedObject.name** (string), required
+
+        Name of the referent; More info: http://kubernetes.io/docs/user-guide/identifiers#names
+      -->
+      
+      - **metrics.object.describedObject.name** (string)，必需
+      
+        被引用对象的名称；更多信息： http://kubernetes.io/docs/user-guide/identifiers#names
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:652]:
```suggestion
        被引用对象的名称；更多信息： http://kubernetes.io/zh-cn/docs/user-guide/identifiers#names
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 16]
```diff
@@ -0,0 +1,2359 @@
...
+      
+    <!--
+    - **metrics.object.metric** (MetricIdentifier), required
+
+      metric identifies the target metric by name and selector
+    -->
+    
+    - **metrics.object.metric** (MetricIdentifier)，必需
+
+      metric 通过名称和选择器标识目标指标。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:672]:
```suggestion
      metric 通过名称和选择算符识别目标指标。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 17]
```diff
@@ -0,0 +1,2359 @@
...
+    
+      <!--
+      *MetricIdentifier defines the name and optionally selector for a metric*
+
+      - **metrics.object.metric.name** (string), required
+
+        name is the name of the given metric
+      -->
+      
+      **MetricIdentifier 定义指标的名称和可选的选择器**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:684]:
```suggestion
      **MetricIdentifier 定义指标的名称和可选的选择算符**
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 18]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **metrics.object.metric.selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)
+
+        selector is the string-encoded form of a standard kubernetes label selector for the given metric When set, it is passed as an additional parameter to the metrics server for more specific metrics scoping. When unset, just the metricName will be used to gather metrics.
+      -->
+      
+      - **metrics.object.metric.selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)
+
+        selector 是给定指标的标准 Kubernetes 标签选择器的字符串编码形式。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:698]:
```suggestion
        selector 是给定指标的标准 Kubernetes 标签选择算符的字符串编码形式。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 19]
```diff
@@ -0,0 +1,2359 @@
...
+      - **metrics.object.target.type** (string), required
+
+        type represents whether the metric type is Utilization, Value, or AverageValue
+      -->
+      
+      **MetricTarget 定义特定指标的目标值、平均值或平均利用率**
+
+      - **metrics.object.target.type** (string)，必需
+
+        type 表示指标类型是 Utilization、Value 或 AverageValue。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:726]:
```suggestion
        type 表示指标类型是 `Utilization`、`Value` 或 `AverageValue`。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 20]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **metrics.object.target.averageUtilization** (int32)
+
+        averageUtilization is the target value of the average of the resource metric across all relevant pods, represented as a percentage of the requested value of the resource for the pods. Currently only valid for Resource metric source type
+      -->
+      
+      - **metrics.object.target.averageUtilization** (int32)
+
+        averageUtilization 是 Pod 所有相关的资源指标平均值的目标值，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:736]:
```suggestion
        averageUtilization 是跨所有相关 Pod 得出的资源指标均值的目标值，
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 21]
```diff
@@ -0,0 +1,2359 @@
...
+        表示为 Pod 资源请求值的百分比。目前仅对资源指标源类型有效。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:737]:
```suggestion
        表示为 Pod 资源请求值的百分比。目前仅对 “Resource” 指标源类型有效。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 22]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **metrics.object.target.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue is the target value of the average of the metric across all relevant pods (as a quantity)
+      -->
+      
+      - **metrics.object.target.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue 是 Pod 所有相关指标平均值的目标值（作为数量）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:747]:
```suggestion
        averageValue 是跨所有 Pod 得出的指标均值的目标值（以数量形式给出）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 23]
```diff
@@ -0,0 +1,2359 @@
...
+
+      <!--
+      - **metrics.object.target.value** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        value is the target value of the metric (as a quantity).
+      -->
+      
+      - **metrics.object.target.value** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        value 是指标的目标值（作为数量）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:757]:
```suggestion
        value 是指标的目标值（以数量形式给出）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 24]
```diff
@@ -0,0 +1,2359 @@
...
+
+  <!--
+  - **metrics.pods** (PodsMetricSource)
+
+    pods refers to a metric describing each pod in the current scale target (for example, transactions-processed-per-second).  The values will be averaged together before being compared to the target value.
+  -->
+  
+  - **metrics.pods** (PodsMetricSource)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:765]: mark

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 25]
```diff
@@ -0,0 +1,2359 @@
...
+
+  <!--
+  - **scaleTargetRef.name** (string), required
+
+    Name of the referent; More info: http://kubernetes.io/docs/user-guide/identifiers#names
+  -->
+
+  - **scaleTargetRef.name** (string)，必需
+
+    被引用对象的名称；更多信息： http://kubernetes.io/docs/user-guide/identifiers#names
```
[the remark this answers is not in the document]
Comment (zhangxyjlu) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:119]: 此连接无效：http://kubernetes.io/zh-cn/docs/user-guide/identifiers#names
是否修改为：https://kubernetes.io/zh-cn/docs/concepts/overview/working-with-objects/names/#names

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md (hunk 15 above)
Comment (zhangxyjlu) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:653]: 此连接无效：http://kubernetes.io/zh-cn/docs/user-guide/identifiers#names
是否修改为：https://kubernetes.io/zh-cn/docs/concepts/overview/working-with-objects/names/#names

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 26]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **currentMetrics.pods.current.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue is the current value of the average of the metric across all relevant pods (as a quantity)
+      -->
+      
+      - **currentMetrics.pods.current.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        verageValue 是跨所有相关 Pod 的指标均值的当前值（以数量形式给出）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:1476]:
```suggestion
        averageValue 是跨所有相关 Pod 的指标均值的当前值（以数量形式给出）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 27]
```diff
@@ -0,0 +1,2359 @@
...
+  
+    <!--
+    *ResourceMetricStatus indicates the current value of a resource metric known to Kubernetes, as specified in requests and limits, describing each pod in the current scale target (e.g. CPU or memory).  Such metrics are built in to Kubernetes, and have special scaling options on top of those available to normal per-pod metrics using the "pods" source.*
+
+    - **currentMetrics.resource.current** (MetricValueStatus), required
+
+      current contains the current value for the given metric
+    -->
+    
+    **ResourceMetricSource 指示如何根据请求和限制中指定的 Kubernetes 已知的资源指标进行扩缩容，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:1548]:
```suggestion
    ResourceMetricSource 指示如何根据请求和限制中指定的 Kubernetes 已知的资源指标进行扩缩容，
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 28]
```diff
@@ -0,0 +1,2359 @@
...
+    此结构描述当前扩缩目标中的每个 Pod（例如 CPU 或内存）。在与目标值比较之前，这些值将被一起平均。
+    此类指标内置于 Kubernetes 中，并且在使用 “Pods” 源的、按 Pod 统计的普通指标之外支持一些特殊的扩缩选项。**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:1550]:
```suggestion
    此类指标内置于 Kubernetes 中，并且在使用 “Pods” 源的、按 Pod 统计的普通指标之外支持一些特殊的扩缩选项。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 29]
```diff
@@ -0,0 +1,2359 @@
...
+  
+      <!--
+      - **currentMetrics.resource.current.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue is the current value of the average of the metric across all relevant pods (as a quantity)
+      -->
+      
+      - **currentMetrics.resource.current.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        verageValue 是跨所有相关 Pod 的指标均值的当前值（以数量形式给出）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:1581]:
```suggestion
        averageValue 是跨所有相关 Pod 的指标均值的当前值（以数量形式给出）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md [hunk 30]
```diff
@@ -0,0 +1,2359 @@
...
+      
+      <!--
+      - **currentMetrics.external.current.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        averageValue is the current value of the average of the metric across all relevant pods (as a quantity)
+      -->
+      
+      - **currentMetrics.external.current.averageValue** (<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+        verageValue 是跨所有相关 Pod 的指标均值的当前值（以数量形式给出）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:1229]:
```suggestion
        averageValue 是跨所有相关 Pod 的指标均值的当前值（以数量形式给出）。
```

**Comment by zhangxyjlu** [comment]
已修改完检视出的所有问题，感谢指导@[tengqm](https://github.com/tengqm)

**Comment by windsonsea** [comment]
The preview is all right. Thanks.

/lgtm

**Comment by Sea-n** [comment]
Hey @zhangxyjlu,

Just reminding, please avoid use `fixes #xxx` for umbrella issues.
We can use `related: #xxx` or `part of: #xxx` so that the umbrella issue would not be closed unintentionally.

**Pull request merged**